C7/c7-static-map-save.json

```diff
@@ -67224,111 +67273,167 @@
         "baseFoodProduction": 0,
         "baseShieldProduction": 1,
         "baseCommerceProduction": 0,
-        "movementCost": 1
+          "movementCost": 1,
+	  "defenseBonus": {
+	      "description": "Desert",

```

Comment:
Why does defenseBonus have its own description instead of using displayName?

Reply:
That was the easiest way to do it. As an alternative we could save only the defense bonus amounts and fill in the descriptions automatically at load time. We'll need to do another pass over descriptions in any case to make them translatable. They might become text keys and/or be auto-genenerated. For now I think it's fine to just store them as plain strings.